core/src/components/modal/modal.tsx

```diff
@@ -1048,6 +1052,10 @@ interface ModalOverlayOptions {
    * to fade in when using a sheet modal.
    */
   backdropBreakpoint: number;
+  /**
+   * The theme used by the sheet modal
+   */
+  theme: Theme;
```

Comment:
Instead of passing `theme`, it might be best to pass `initialBackdrop` and `backdropValue`. These two new values would allow any theme to be customizable instead of relying on checking for specific themes.